pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic
+from sympy.physics.mechanics import ReferenceFrame, Particle, RigidBody, dynamicsymbols
+
+# local
+from ..bodies import Body, Ground
+from ..force import Force
+
+class TestBody():
+    def setup(self):
+        self.body = Body('Body')
+
+    def test_body_init(self):
+        # by default a rigidbody is created.
+        assert self.body.name == 'Body'
+        assert self.body.parent is None
+        assert self.body.child is None
```

Comment:
Why do parent child relationships need to be stored on body? I thought this was stored on Joint. I don't think it is necessary to store it both on Joint and Body.